lib/src/session.dart

```diff
@@ -233,16 +228,11 @@ class SessionConnectionStateController {
     if (_token != null) {
       throw RealmStateError("Session connection state subscription already started");
     }
-
     _token = realmCore.sessionRegisterConnectionStateNotifier(_session, this);
   }
 
   void _stop() {
-    if (_token == null) {
-      return;
-    }
-
-    realmCore.sessionUnregisterConnectionStateNotifier(_session, _token!);
```

Comment:
Are we sure we don't need to unregister anymore? Or perhaps, that is the new change in Core?

Reply:
Yes, they changed it. Now you `realm_release` a type of notification token instead

Reply:
makes sense